airflow/www/static/js/dag/details/index.tsx

```diff
@@ -136,7 +144,11 @@ const Details = ({
   const isMappedTaskSummary = isMapped && mapIndex === undefined && taskId;
   const isGroup = !!children;
   const isGroupOrMappedTaskSummary = isGroup || isMappedTaskSummary;
-  const showLogs = !!(isTaskInstance && !isGroupOrMappedTaskSummary);
+  const isIndividualTaskInstance = !!(
+    isTaskInstance && !isGroupOrMappedTaskSummary
+  );
+  const showLogs = isIndividualTaskInstance;
+  const showXcom = isIndividualTaskInstance;
```

Comment:
We can just use `isIndividualTaskInstance` instead of creating new vars. Also, we can drop `individual` for brevity's sake.

Reply:
Although we should refactor the `isTaskInstance` var to reduce confusion.

Remove `isInvididualTaskInstance` and `isGroupOrMappedTaskSummary`.
Then just have `const isTaskInstance = taskId && runId && !isGroup && !isMappedSummary`

Reply:
I have tried to refactor this as per your suggestion in commit [fec1b](https://github.com/apache/airflow/pull/35719/commits/fec1baedc812921bae3b621eb8dc9a9782f5817f). I'm not too sure if this is what you intended.

I also removed the `show...` vars and use the `is...` vars for rendering since they are just one to one. If this makes thing more confusing I can easily revert